### DISCUSSION TITLE
How to get actual query which is sent to sql database ?

### DISCUSSION THREAD

**Discussion by pravinkottawar** [prisma/prisma#10421] [Q&A] [answered]
Generally we write like following

prisma.table name from schema.prisma..create(data:data)

This is easiest way I found out to insert data into db in Node JS.
My doubt is can we get exact query which is being passed to db ?

**Comment by pantharshit00** [comment, marked as answer]
Hello, 

If you want to see the sql executed, you can enable query logging. Documentation for this feature is here: https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#examples-1 and https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#log

Basically do:
```ts
const prisma = new PrismaClient({ log: ["query"] })
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
